scripts/metadata_table.py

```diff
@@ -993,8 +993,26 @@ def parse_variable(self, curr_line, known_ddts, skip_ddt_check=False):
                             pval = []
                             for dim in porig:
                                 if ':' in dim:
+                                    for dim2 in dim.split(':'):
+                                        dim_ok = VarDictionary.loop_var_okay(standard_name=dim2,
+                                            is_run_phase=self.__section_title.endswith("_run"))
+                                        if not dim_ok:
+                                            emsg = "horizontal dimension"
```

Comment:
Should we make this more informative, maybe "ERROR: horizontal_dimension found in run Phase"

Reply:
This is already included in the context I think. I had more stuff there and then removed it because it was redundant.

Reply:
It's like this, and that provides sufficient information imo:
```
Invalid horizontal dimension, 'horizontal_loop_end', at /home/dom/work/ccpp/ccpp-framework/test/unit_tests/sample_host_files/mismatch_hdim_mod.meta:25
```

Reply:
Looks good to me as well. Context matters